### DISCUSSION TITLE
[Feature Request] Load Theme without instance

### DISCUSSION THREAD

**Discussion by akshaydmc** [microsoft/monaco-editor#3988] [Q&A]
### Context

- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*
- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*

### Description

I have upgraded the version of monaco-editor and monco-webpack-plugin editor to the latest one. 

Themes were managed globally on the `window.monaco` object, but since that doesn’t exist until a code editor is rendered, we used `StaticServices.standaloneThemeService.get();` to register our themes. 

This seems to be unavailable in newer Monaco versions, based on the inability to import `StandaloneServices`, and this comment in [[the source](https://github.com/microsoft/vscode/blob/main/src/vs/editor/standalone/browser/standaloneServices.ts#L1059-L1062)]

So is there any workaround solution in which we can pass our custom theme config to colorize elements?

**Comment by hediet** [comment]
> window.monaco object, but since that doesn’t exist until a code editor is rendered,

I'm not sure that is true. You can already register language features without having to create a code editor!
